filelink.rb

```diff
@@ -0,0 +1,15 @@
+module FileLink
+
+  def file_link.write(name, desg, dept)
+    file_name = "employeedetails.csv"
+    CSV.open(file_name, "a") do |file|
+      file << [name. desg, dept]
+    end
+  end
+
+  def file 
+    @@file_name = "employeedetails.csv"
+    file = csv.parse(File.open(@@file_name, 'r'))
```

Comment:
you don't need the file variable. 
csv.parse... can be the last line. It will return the file from the method